internal/app/agent/service.go

```diff
@@ -186,23 +179,60 @@ func (a *agent) Control(uuid, cmdStr string) error {
 // Example of creation:
 // 	b, _ := toml.Marshal(cfg)
 // 	config_file_content := base64.StdEncoding.EncodeToString(b)
-func (a *agent) ServiceConfig(uuid, cmdStr string) error {
+func (a *agent) ServiceConfig(uuid, cmdStr string) (err error) {
 	cmdArgs := strings.Split(strings.Replace(cmdStr, " ", "", -1), ",")
-	if len(cmdArgs) < 3 {
+	if len(cmdArgs) < 1 {
 		return errInvalidCommand
 	}
+	services := []byte{}
+	resp := ""
+	cmd := cmdArgs[0]
+
+	switch cmd {
+	case "view":
+		services, err = json.Marshal(a.Services())
+		resp = string(services)
+	case "save":
```

Comment:
Please leave cmd example in the command, so that we can understand this parsing.